service/worker/indexer/esProcessor.go

```diff
@@ -191,6 +195,13 @@ func (p *esProcessorImpl) ackKafkaMsgHelper(key string, nack bool) {
 	}
 
 	if nack {
+		wid, rid, domainID, err := p.getMsgInfo(kafkaMsg.message)
+		if err != nil {
+			p.logger.Error("Unable to process message.", tag.WorkflowID(wid), tag.WorkflowRunID(rid),
```

Comment:
How about merging this log with the log on L168, so that in the one log we can have both workflow information and the response from es? I think it will make investigation a little bit easier.